### DISCUSSION TITLE
Select window option in the back menu key presses

### DISCUSSION THREAD

**Discussion by NASTYWORKZ** [ClassicOldSong/moonlight-android#12] [Ideas]
Add CTRL+ALT+TAB to the key press menu for games that open in an out of focus window. Also useful for switching to another window for any reason.

**Comment by ClassicOldSong** [comment]
You can use https://github.com/Julow/Unexpected-Keyboard for more combination keys and customizable keyboard layout for now.

**Reply by NASTYWORKZ** [reply]
I use an Nvidia Shield with a remote so I don't think this would work as expected. Thanks anyway :)

**Comment by ClassicOldSong** [comment]
It's now added: https://github.com/ClassicOldSong/moonlight-android/releases/tag/v12.1-artemis-240911

You can also set server commands with the latest [Apollo](https://github.com/ClassicOldSong/Apollo/releases) to execute more convenient commands.

**Reply by NASTYWORKZ** [reply]
Thank you, this support is so good actually adding requested features that got ignored with "alternatives"!
